modules/seqexec/web/client/src/main/scala/seqexec/web/client/model/ModelOps.scala

```diff
@@ -3,14 +3,21 @@
 
 package seqexec.web.client
 
-import seqexec.model.Model.{Resource, Instrument, SequenceState, SequenceView, Step, StepState, StandardStep}
 import cats.Show
 import cats.implicits._
+import cats.data.NonEmptyList
+import gem.enum.Site
+import seqexec.model.Model.{Resource, Instrument, SequenceState, SequenceView, Step, StepState, StandardStep}
 
 /**
   * Contains useful operations for the seqexec model
   */
 object ModelOps {
+  implicit val siteStateShow: Show[Site] = Show.show[Site] {
```

Comment:
@tpolecat Why there is no `Show[Site]` Is it because sometimes you want `GN/GS` and sometimes the full name?

Reply:
if `Show` is always `toString` what's the point of having them? I guess I can just use `site.shortName` then

Reply:
`Show` isn't very useful because very few things have a canonical string representation; and those that do often have an associated parser, which `Show` says nothing about. So I just see it as a better `toString`; better in the sense that you have to opt in.

Reply:
There is a `Display` typeclass in `Enumerated.scala` that we might revisit. There are no instances, I was just trying to sketch out what a typeclass for UI elements might look like.

Reply:
hmm, I don't see why would it be better than a plain to`toString` if we delegate. I see the point of the parsing and having better ways to display but if `Show` is just a glorified `toString` I'd rather remove them and save us the extra code it requires

Reply:
It's better in the sense that not everything has a useful `toString` (functions, for example) so `Show` prevents you from trying to print out things that will be unreadable. But the larger principle is that `toString` and `hashCode` and `equals` and other methods defined on `Any` break parametricity and make our types less meaningful, so we should avoid using them.

But the case for `toString` is weak since it's not that useful to begin with. It wouldn't bother me that much to get rid of the instances, but we should wait and chat with @swalker2m about it.